sdk/src/Handlers/AspNet/AWSXRayRecorder.Handlers.AspNet.csproj

```diff
@@ -30,7 +30,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.Core" Version="3.3.25.1" />
+    <PackageReference Include="AWSSDK.Core" Version="3.5.1.23" />
```

Comment:
Do we need to depend on AWSSDK.Core for this project? If not, can we remove it?

Reply:
Yes, we need the `Logger` from `AWSSDK.Core` at this moment

Reply:
From what I understand, if the AspNet project references the `AWSXRayRecorder.Core` project which uses AWSSDK.Core, this project should be able to use the `Logger` from AWSSDK.Core as transitive dependency?

Reply:
Nice point, updated